docs/contributing.rst

```diff
@@ -184,8 +184,10 @@ To release a new version of the theme, core team will take the following steps:
 #. Update the changelog (``docs/changelog.rst``) with the version information.
 #. Run ``python setup.py update_translations`` to compile new translation files
    and update Transifex.
-#. Run ``python setup.py build_assets`` to rebuild all the theme assets and the Python
-   package.
+#. Run ``npm run build`` to rebuild all the theme assets and the Python
```

Comment:
`python setup.py build_assets` does more than `npm run build`, so I would leave these directions for now.

Reply:
Can you explain? I see it running `subprocess.run(['node_modules/.bin/webpack', '--config', 'webpack.prod.js'], check=True)` - isn't that the same as `npm run build`?

Reply:
It should also run `python setup.py build`, as it's an extension of the build command.

Reply:
Ah, roger, will fix..

Reply:
Ah I see, this was changed. With https://github.com/readthedocs/sphinx_rtd_theme/pull/1039/files

But `python setup.py build_assets` also does `npm install` to update the lock file.

I'd say the original is still correct and we shouldn't change it yet.

Reply:
I did also like getting rid of the custom commands so we can get rid of `setup.py` all-together...

Reply:
Because these are currently working, this would be a good change to leave for later.